globals.go

```diff
@@ -172,9 +172,24 @@ const (
 	MaxDurationPerDistroHost               = 30 * time.Minute
 	MaxDurationPerDistroHostWithContainers = 2 * time.Minute
 
-	// max expirations
-	SpawnHostExpireDays = 30
-	HostExpireDays      = 10
+	// Spawn hosts
+	SpawnHostExpireDays                 = 30
+	HostExpireDays                      = 10
+	ExpireOnFormat                      = "2006-01-02"
+	DefaultMaxSpawnHostsPerUser         = 3
+	DefaultSpawnHostExpiration          = 24 * time.Hour
+	SpawnHostNoExpirationDuration       = 7 * 24 * time.Hour
+	MaxSpawnHostExpirationDurationHours = 24 * time.Hour * 14
+
+	// ec2 resource tag names
+	TagName             = "name"
```

Comment:
These would make more sense to go in the cloud package, since they don't apply to other cloud providers

Reply:
The tags ended up here because the `TagExpireOn` tag is used in the host package and importing the cloud package from the host package caused a circular dependency.
I could hear an argument that the tags themselves are independent of a particular cloud provider because they're a read by the reaper and they're stored in the host document.

Reply:
Fair, could you adjust the comment then?

Reply:
done